chaincode/output.go

```diff
@@ -279,12 +279,13 @@ type outputModel struct {
 	Testtuple           outputTesttuple            `json:"testtuple"`
 }
 
-// TuplesEvent is the collection of tuples sent in an event
-type TuplesEvent struct {
+// Events is the collection of tuples sent in an event
+type Events struct {
```

Comment:
Is there a particular reason to add an s to the struct name? For me, having a "single" event with multiple things inside is easier to reason with. But maybe that's just me!

Reply:
I add an internal struggle here. I agree that there is in fact only one event from hyperledger point of vue and only one structure but each item in this struct's slices is by itself an event.
But I would be ok with `Event`. @inalgnu @samlesu any strong feeling about that?